lib/view_component/base.rb

```diff
@@ -282,19 +290,31 @@ def content_evaluated?
     mattr_accessor :test_controller
     @@test_controller = "ApplicationController"
 
-    # Set if render monkey patches should be included or not in Rails <6.1:
-    #
-    #     config.view_component.render_monkey_patch_enabled = false
-    #
-    mattr_accessor :render_monkey_patch_enabled, instance_writer: false, default: true
+    if Rails.version.to_f >= 5.2
```

Comment:
You're right, but the [newer syntax](https://github.com/rails/rails/blob/2459c20afb508c987347f52148210d874a9af4fa/activesupport/lib/active_support/core_ext/module/attribute_accessors.rb#L191-L195) is with the `default:` keyword argument, that's why I went with it.

Either way if fine with me.

Reply:
The nice thing about the block approach is that it's only called if no value has been provided, meaning it could potentially cut down on unnecessary object allocations. I can't imagine it'd save that many, but still worth considering.